plugin/helpers_bots.go

```diff
@@ -10,6 +10,12 @@ import (
 )
 
 func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error) {
+	const minimumSupportedVersion = "5.10"
+
+	if p.API.GetServerVersion() < minimumSupportedVersion {
```

Comment:
I don't think it's as easy. Does this simple check handle all cases? I'm inclined to sugesst using https://github.com/blang/semver just to make sure we don't miss a case

Reply:
Yes, https://play.golang.org/p/WTBTE-YJORm. I did a small test case sample. it fails. It's not simple. Let me use semver.